proposals/wg-reference-spec.md

```diff
@@ -0,0 +1,82 @@
+# OCI Working Group: Reference Spec
+
+## Reference Spec OCI Working Group - Governance Charter
+
+This document describes the basic governance principles for the Reference Spec Working Group (the “WG”).
+
+The WG operates as an OCI Working Group under the [Open Container Initiative (OCI) Charter](https://github.com/opencontainers/tob/blob/master/CHARTER.md), which describes the responsibilities of the OCI Technical Oversight Board (the "TOB”). The WG is established by the TOB as an OCI Working Group pursuant to the OCI Charter. Accordingly, the WG will operate in accordance with the OCI Charter and OCI's other policies and procedures, supplemented by the details below.
+
+## Purpose
+
+References are a string that is used by runtimes and other OCI registry clients to retrieve a container image.
+They are currently a convention, used by many clients, adopted from Docker and implemented in distribution/distribution.
+This WG seeks to define the syntax and parsing of a reference as an OCI spec.
+
+## Scope
+
+* Document the existing convention used by runtimes and other OCI registry clients.
+* Specify the syntax for references to an [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/7b36cea86235157d78528944cb94c3323ee0905c/image-layout.md) manifest, including support for a tag, digest, or a full registry/repository name.
+* Define how the syntax can be extended to support other use cases, including:
+  * Alternate layer formats
+  * Immutable tags
+  * Selecting an artifact that refers to another manifest (e.g. signature or SBOM)
+  * Querying content from a pull through cache serving multiple registries (see [distribution-spec PR #66](https://github.com/opencontainers/distribution-spec/pull/66))
+* Provide backwards compatibility by using the existing reference convention from [distribution/reference](https://github.com/distribution/reference) when feasible.
```

Comment:
This increases my confusion -- we're trying to talk about "backwards compatibility" but we're explicitly trying to design a whole new thing?

Reply:
We're defining something that can be an in-place upgrade for tooling that currently uses the distribution/reference strings. v1.0 may end up being the current state of distribution/reference, but v* should avoid changes that break users who depend on the current string syntax. New features should be additive in a way that can be distinguished from current use cases, without ambiguity.